app/views/content/life-as-a-teacher/explore-subjects/languages/_article.html.erb

```diff
@@ -0,0 +1,150 @@
+  <p>Find out <a href="https://www.gov.uk/government/publications/gcse-ancient-foreign-languages">what pupils will learn when studying ancient languages at GCSE</a>.</p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading--box-blue">Start your journey into teaching languages</h2>
+    <h3>Check your qualifications</h3>
+    <p>To train to teach in secondary schools in England, you’ll need:</p>
+    <ul>
+      <li>GCSEs at grade 4 (C) or above in English and maths</li>
+      <li>a bachelor’s degree in any subject</li>
+    </ul>
+    <p>Having relevant A levels can show your subject knowledge, if you do not have a language degree.</p>
+    <p>Language teacher training courses have had successful applications from candidates with a range of degrees, including translation studies, English language and literature, psychology and education studies.
+    <p>Find out more about what <a href="/train-to-be-a-teacher/qualifications-you-need-to-teach">qualifications you need to train to teach</a>.</p>
+    <%= render Content::ExpanderComponent.new(
+      title: "check your qualifications",
+      text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
+      link_title: "You can get help comparing English and international qualifications.",
+      link_url: "/non-uk-teachers/non-uk-qualifications",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Subject knowledge enhancement course (SKE)</h3>
+    <p>If your training provider thinks you need to top up your subject knowledge in languages before you become a teacher, you can do a subject knowledge enhancement (SKE) course.
+    <p>There are lots of reasons why someone might need to complete an SKE course. For example, if you finished your degree more than 5 years ago or if you have an A level but not a degree in the subject you want to teach.</p>
+    <p><a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">Find out more about subject knowledge enhancement courses</a>.</p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p>Tax-free bursaries of <%= v :bursaries_postgraduate_languagesother %> are available for all trainee language teachers.</p>
+    <p>Scholarships of <%= v :scholarships_languagesfrenchgermanspanish %> are available for eligible trainee French, German or Spanish language teachers.</p>
+    <p>You can <a href="/funding-and-support/scholarships-and-bursaries">get a bursary or scholarship</a> alongside a tuition fee loan and maintenance loan.</p>
+    <p>You may also be able to get <a href="/funding-and-support">extra funding and support</a> if you're:</p>
```

Comment:
Should we remove the link here now that we have 'Find out how you can fund your teacher training' below?